docs/zuri_app(desktop).md

```diff
@@ -0,0 +1,239 @@
+# Zuri Chat App (Desktop)
+## `Overview`
+ZuriChat is a collaboration hub that brings the right people and tools together to get work done. From Fortune 100 companies to corner markets, millions of people around the world use ZuriChat to connect their teams, unify their systems, and drive their business forward.
+
+## `Goals`
+- Provide a platform for team colaboration
+- Make colaboration easy and seamless
+- Make modern workplace fun
+- A user friendly app for colaboration
+  
+## `Requirements`
+- DM,Chats and Channels should be working
+- All plugins should be functional
+- Users should be able to create and join workspaces
+- Users should be able to add and remove plugins seamlessly
+  
+## `Open Questions`
+- How do we manage system updates across teams
+- How can the app support multiple languages and regions
+- How the we serve companies who aren't tech savy
+
+## `Approach`
+- We will work with team leads of other departments to ensure we update with thier works and vice versa?
```

Comment:
The spelling of their

Reply:
Use a present continuous sentence like "we are"  or 'we are in the process of'. Using " we will" feel like you haven't started working.

Reply:
done